src/scripting/wrapper.hpp

```diff
@@ -3,8 +3,8 @@
  *  'src/scripting/wrapper.interface.hpp'
  * DO NOT CHANGE
  */
-#ifndef HEADER_SUPERTUX_SCRIPTING_WRAPPER_HPP
-#define HEADER_SUPERTUX_SCRIPTING_WRAPPER_HPP
+#ifndef HEADER_SUPERTUX_SQUIRREL_WRAPPER_HPP
```

Comment:
Why was this changed?

Reply:
"SCRIPTING" was inconsistent when using Miniswig in other projects. I settled for using "SQUIRREL" because it'd fit in all scenarios, but it could also be made so it's set by an argument.

Reply:
I thought the include guards were meant to follow the file's pathname?

Reply:
That's true. I could make it so it generates the header based on the path in `--output-hpp-include`.